src/Planner/CollectTableExpressionData.cpp

```diff
@@ -96,12 +96,12 @@ void collectTableExpressionData(QueryTreeNodePtr & query_node, PlannerContext &
 
         if (auto * table_node = table_expression_node->as<TableNode>())
         {
-            bool storage_is_remote = table_node->getStorage()->isRemote();
+            bool storage_is_remote = table_node->getStorage()->isRemote(planner_context.getQueryContext());
```

Comment:
I tried to find out where `isRemote()` is used for a new analyzer. Looks like it is not used yet, cause distributed processing is not written. But I hope it would be similar to what we have currently.